apps/server/src/modules/authentication/controller/dto/change-password.param.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString, Matches } from 'class-validator';
+import { passwordPattern } from './password-pattern';
+
+export class ChangePasswordParam {
```

Comment:
This should be plural like `ChangePasswordParams` and `change-password.params.ts` by convention even if the dto contains just one parameter.